packages/fxa-settings/src/components/App/index.tsx

```diff
@@ -29,7 +29,17 @@ export const App = ({ queryParams }: AppProps) => {
             }
             hasSubscription={Boolean(account.subscriptions.length)}
           >
-            <Settings {...{ account }} />
+            <Suspense fallback="">
```

Comment:
Not really sure what the fallback here should be?

Reply:
what are we suspending on here?

Reply:
The payments server uses it for loading the router routes, https://github.com/mozilla/fxa/blob/9592b42337a75fdddefebbcf579408b7254be26b/packages/fxa-payments-server/src/App.tsx#L101. In my testing there seems to be a loading indicator on the page, but can't track down where/how that gets into the page. Maybe Suspense is not needed?

Reply:
I don't _think_ it's needed. our spinner is in AccountDataHOC

Reply:
If the content inside `FlowContainer` hangs for some reason I think we could use the spinner here, but I don't know that would happen in Settings because we're not querying anything in the flows, and even if we were, GQL provides that `loading` check that yeah in AccountDataHOC we use to display a loading spinner so I don't _think_ a fallback here would ever be shown.